torchvision/prototype/datapoints/_image.py

```diff
@@ -23,6 +23,11 @@ def __new__(
         device: Optional[Union[torch.device, str, int]] = None,
         requires_grad: Optional[bool] = None,
     ) -> Image:
+        if isinstance(data, PIL.Image.Image):
+            from ..transforms import functional as F
```

Comment:
isn't this also accessible `self._F`?

Reply:
In theory yes, but inside `__new__` we don't have `self` yet.

Reply:
It's a classmethod and even `__F` can be None if this is the first call, so I prefered to import